docs/parts/imu.md

```diff
@@ -72,6 +69,28 @@ Valid settings are from 0 to 6:
 - `5` 10Hz
 - `6` 5Hz
 
+<br>
+
+2. In `/home/pi/projects/donkeycar/donkeycar/parts/` find `imu.py`
```

Comment:
I've actually landed a pull requests that exposes a configuration so there is not need to edit this manually; just change the configuration value; See https://github.com/autorope/donkeycar/pull/1064.  The configuraton changes here https://github.com/autorope/donkeycar/pull/1064/files#diff-dd7806493c6284e60142951760a6712d8b3f90a8e007e710d4033691f74753edR447.  So it would be excellent if you could incorporate instructions for changing that configuration here.

Reply:
Tried to understand what's happening at the provided recent merge.
==> configurable imu addressing (?); derives from imu part file w/ new (?) IMU_ADDRESS config (?)

Thus, and please do correct me, @Ezward 

1) the imu docs should instead reflect setting `addr=0x68` to `addr=IMU_ADDRESS` on line `29`
... in addition to commenting out the `calibrateMPU6500()` script
... & setting `address_mpu_slave = 0x0c`
... not sure where `IMU_ADDRESS` would be initially defined here

_Or_
 
2) are you suggested the imu docs should additionally be extended to contain info on how to modify:
 `IMU_ADDRESS = 0x68              # if AD0 pin is pulled high them address is 0x69, otherwise it is 0x68` 
...which can be found at: `donkeycar/templates/cfg_path_follow.py` (?)

Reply:
I think option 2 seems more likely.
So, I should include instructions how to configure `IMU_ADDRESS` in:

1. `donkeycar/templates/cfg_path_follow.py::414`

_&_

3. `donkeycar/templates/cfg_complete.py::447`

Appreciate the feedback

Reply:
See https://github.com/autorope/donkeycar/blob/f8dc720596d04c3cb14ff0a34d771ef39baf1897/donkeycar/templates/complete.py#L263 https://github.com/autorope/donkeycar/blob/f8dc720596d04c3cb14ff0a34d771ef39baf1897/donkeycar/templates/complete.py#L805  The imu part has be updated to take in the necessary parameters and complete.py has been updated to use them.  

I also see that the instruction include setting the `address_mpu_slave=0x0c`; can you explain?  My understanding what that the slave setting is if you have two IMU's and one is slaved to the other.  

I also see that the instructions include commenting out the sensor calibration; can you explain?  The calibrateMPU6500 is still appropriate even if using an MPU9250; the MPU9250  is basically just an MPU6500 plus an magnetometer; we we still want to calibrate the giro/accelerometers so we know which way is down.